bin/fuel-convert

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+import argparse
+import importlib
+import pkgutil
+
+from fuel import converters
+
+
+if __name__ == "__main__":
+    dataset_options = [
+        m[1] for m in pkgutil.iter_modules(converters.__path__) if not m[2]]
+    parser = argparse.ArgumentParser()
+    parser.add_argument("dataset", help="what dataset to convert",
+                        choices=dataset_options)
+    parser.add_argument("-d", "--directory", help="directory in which input " +
```

Comment:
I think it'd be nice if the directory defaulted to `os.getcwd()`, and the output file to `os.path.join(os.getcwd(), '{}.hdf'.format(dataset))`. So if I downloaded MNIST I can just run `fuel-convert mnist` in the directory I downloaded it to to get `mnist.hdf5`.